Neos.ContentRepository.Core/Classes/Feature/NodeModification/Dto/SerializedPropertyValue.php

```diff
@@ -38,8 +38,6 @@ private function __construct(
     }
 
     /**
-     * If the value is NULL an unset-property instruction will be returned instead.
```

Comment:
unrelated and otherwise not changed file

Reply:
I know yeah that the comment is false was my fault and i fixed it ^^